src/yaml-support/yaml-util.ts

```diff
@@ -122,4 +122,39 @@ function getStringValue(node, key: string, ignoreCase: StringComparison = String
             }
         }
     }
-}
\ No newline at end of file
+}
+
+// copied from explainer.ts, used by explainer.ts, yaml-schema.ts
```

Comment:
I'd avoid putting these in yaml-util as they are also used for JSON files (Explain works on JSON as well as YAML).  I think `explainer.ts` is still a reasonable place for them because that module is meant to be a reasonably pure 'transform Swagger to Markdown' module and reusable outside the hover provider; but if you're not comfortable with that then perhaps create a new module called something like `schema-formatting`?  In which case you could arguably rename these functions to e.g. `formatOne`, `formatComplex`, etc.